templates/vanilla/packages/client/src/mud/setupNetwork.ts

```diff
@@ -1,18 +1,52 @@
+/* Definitions required to connect to a blockchain */
+
+/*
+ * The MUD client code is built on top of Viem
```

Comment:
```suggestion
 * The MUD client code is built on top of viem
```
viem prefers lowercase